test/builtin-storage-class.ll

```diff
@@ -0,0 +1,22 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
Please add comment describing purpose of this test.

Reply:
Done thanks